### ISSUE TITLE
Combining models of different types in a collection

### ISSUE THREAD

**Issue by jamesdixon** [jsonapi-serializer/jsonapi-serializer#183]
Hi there,

Is there a preferred method to combine multiple models of different types in a collection?

I have two models, `Inspection` and `Event`, and need to return those as a single collection. I was expecting that when calling `render jsonapi: models`, that it would automatically pick the proper serializer based on the type of the model but it blows up.

Currently, I'm manually handling the process with this but it feels messy:

```
    inspections = load_inspections
    events = load_events
    serialized_inspections = Api::V1::Mobile::InspectionSerializer.new(inspections).serializable_hash
    serialized_events = Api::V1::Mobile::EventSerializer.new(events).serializable_hash

    eventables = { data: serialized_events[:data] + serialized_inspections[:data] }

    render json: eventables.to_json, content_type: 'application/vnd.api+json'
```

In a perfect world, I'd love to be able to do something like:

```
    inspections = load_inspections
    events = load_events
    
    render jsonapi: inspections + events
```

Thanks and apologies in advance if I'm missing something obvious!

**Comment by stas** [comment]
Thanks @jamesdixon 

I believe what you're describing is #121 
Unfortunately, this will be available only after #141 is merged.

**Comment by stas** [comment]
Duplicate of #121

**Comment by jrochkind** [comment]
I am interested in this feature!

Do I understand correctly that: PR is not feasible/welcome for this against 2.x;  it is planned for 3.x; but there is no ETA and currently no work going on on 3.x?

if so, alternately, does anyone have a monkey-patch that succesfully adds this to 2.x? Some of the discussion above seemed to imply that perhaps someone did?
